build_helper.sh

```diff
@@ -58,9 +58,6 @@ function prepare_upload_data {
   rm -rf ${LOCAL_PATH}
   mkdir -p ${LOCAL_PATH}
   mv handler.zip ${LOCAL_PATH}/lambda.zip
-}
-
-function prepare_cloudformation_code {
```

Comment:
Not sure to follow, if the function is never called, why not removing all of its content?
Also why having build_helper.sh? I feel like the code of the script, could easily go in the Makefile